### PR TITLE
Docs: split v1 and v2

### PR DESCRIPTION
**Pull request by sleepysaurus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -600,6 +600,34 @@ In Chrome, if the origin script tag and remote domain do not meet these requirem
...
+
+You can hook failed Ajax requests with $http in AngularJS by providing an Interceptor that sends to Raygun on error. One possible simple implementation using custom data:
+
+```javascript
+$httpProvider.interceptors.push(function($q, dependency1, dependency2) {
+  return {
+   'requestError': function(rejection) {
+       rg4js('send', {
+          error: 'Failed $http request', 
+          customData: [{ rejection: rejection}]
```
Comment (fundead) [inline README.md]: Don't need the array here, just an object with rejection inside it

File: README.md [hunk 2]
```diff
@@ -600,6 +600,34 @@ In Chrome, if the origin script tag and remote domain do not meet these requirem
...
+       rg4js('send', {
+          error: 'Failed $http request', 
+          customData: [{ rejection: rejection}] 
+       });
+    },
+
+    'responseError': function(rejection) {
+       rg4js('send', {
+          error: 'Failed $http response', 
+          customData: [{ rejection: rejection}]
```
Comment (fundead) [inline README.md]: As above

File: V1Documentation.md [hunk 3]
```diff
@@ -1,53 +1,52 @@
...
 
 **Step 2**
 
 Add the following lines to your JS site code and paste in your API key (from your Raygun dashboard), to set up the provider to automatically send errors to your Raygun:
 
 ```javascript
 <script type="text/javascript">
-    rg4js('apiKey', 'paste_your_api_key_here');
-    rg4js('enableCrashReporting', true);
+  Raygun.init('yourApiKey', { // Other setup options here});
```
Comment (fundead) [inline V1Documentation.md]: That comment won't produce a syntactically valid line with the brackets after it, suggest there's a note with a link to this section: https://github.com/MindscapeHQ/raygun4js/pull/230/files#diff-10512c1bf63d296c6eb4e2faf3de6a24R166

File: V1Documentation.md [hunk 4]
```diff
@@ -1,53 +1,52 @@
...
 </script>
 ```
 
 This will configure the provider to send to your Raygun app, and to automatically send all unhandled errors.
 
 That's it for the basic setup! See **Usage** below for more info on how to send errors.
 
-## Alternative setup options
-
-Note: This library can now be interacted with in two ways, the V1 API and the V2 API. The V1 API is available as 'public' functions on the global Raygun object, and is intended to be used to control the provider during runtime. Legacy setup methods remain on this API for backwards compatibility with 1.x releases. The V2 API is made available when using the snippet (above), and is used to asynchronously configure the provider during onLoad. This is the recommended approach for new setups.
```
Comment (fundead) [inline V1Documentation.md]: This section shouldn't be removed from the V1 docs, but should be reworded to highlight the difference between the V2 version

File: V1Documentation.md [hunk 5]
```diff
@@ -1,53 +1,52 @@
-# Raygun4js
+# Raygun4js V1 Documentation
 
-[Raygun.io][raygunio] provider for client-side JavaScript
+The following documentation is for Raygun4js V1 only. Raygun4js V1 should only be used by legacy customers or for applications that block scripts.
```
Comment (fundead) [inline V1Documentation.md]: should be "for applications that can't use asynchronous loading, such as mobile apps in webviews (Cordova etc)"

File: V1Documentation.md [hunk 6]
```diff
@@ -1,53 +1,52 @@
...
+Go to [V2 documentation][v2docs].
+
+[v2docs]: https://github.com/MindscapeHQ/raygun4js/blob/master/README.md
 
 ## Getting Started
 
 **Step 1**
 
-Add this snippet to your markup, before the closing `</head>` tag:
+Download the [production version][min] or the [development version][max].
```
Comment (fundead) [inline V1Documentation.md]: The CDN should be the preferred option for V1, with the manual download links as an alternate option

File: V1Documentation.md [hunk 7]
```diff
@@ -59,34 +58,37 @@ Run `bower install raygun4js`
...
 
 Visual Studio users can get it by opening the Package Manager Console and typing `Install-Package raygun4js`
 
-#### Manual download
+#### CDN
 
-Download the [production version][min] or the [development version][max]. You can also download a version without
-the jQuery hooks if you are not using jQuery or you wish to provide your own hooks. Get this as a
-[production version][min.vanilla] or [development version][max.vanilla].
+Raygun4JS can also be loaded directly from our content delivery network:
```
Comment (fundead) [inline V1Documentation.md]: Should be preferred setup option as above in Step 1

File: V1Documentation.md [hunk 8]
```diff
@@ -115,51 +113,17 @@ Raygun.detach();
...
-  allowInsecureSubmissions: true
-});
-
-// V1
 Raygun.init('yourApiKey', { allowInsecureSubmissions: true });
 ```
 
-## Documentation
-
-### Differences between Raygun4JS V1 and V2
```
Comment (fundead) [inline V1Documentation.md]: This block should also be present, but reworded to highlight the difference (shortly) for V1

File: V1Documentation.md [hunk 9]
```diff
@@ -199,20 +163,20 @@ objects (for partial matches). Each should match the hostname or TLD that you wa
...
   allowInsecureSubmissions: true,
   ignoreAjaxAbort: true,
   ignoreAjaxError: true,
   debugMode: true,
   ignore3rdPartyErrors: false,
   wrapAsynchronousCallbacks: true,
-  excludedHostnames: ['\.local'],
-  excludedUserAgents: ['Mosaic'],
-  disableCrashReporting: false,
+  excludedHostnames: ['localhost', '\.dev'],
```
Comment (fundead) [inline V1Documentation.md]: Second should be '\.local' as before (should also be that in V2 readme.md if it's not)

File: V1Documentation.md [hunk 10]
```diff
@@ -199,20 +163,20 @@ objects (for partial matches). Each should match the hostname or TLD that you wa
...
+  excludedUserAgents: ['PhantomJS', 'MSIE'],
+  disableErrorTracking: false,
   disablePulse: false,
-  pulseMaxVirtualPageDuration: 1800000,
+  pulseMaxVirtualPageDuration: 1800000
   pulseIgnoreUrlCasing: false
-});
+}).attach();
```
Comment (fundead) [inline V1Documentation.md]: .attach should be on a newline with a comment after it to highlight the fact that it enables CrashReporting

File: V1Documentation.md [hunk 11]
```diff
@@ -453,17 +387,6 @@ Raygun.init('apikey', { disableAnonymousUserTracking: true });
...
-  identifier: 'user_email_address@localhost.local',
-  isAnonymous: false,
-  email: 'emailaddress@localhost.local',
-  firstName: 'Foo',
-  fullName: 'Foo Bar',
-  uuid: 'BAE62917-ACE8-ab3D-9287-B6A33B8E8C55'
-});
-
-// V1
 Raygun.setUser('user_email_address@localhost.local', false, 'user_email_address@localhost.local', 'Foo Bar', 'Foo', 'BAE62917-ACE8-ab3D-9287-B6A33B8E8C55');
```
Comment (fundead) [inline V1Documentation.md]: If we can format each property on newlines like the V2 example with comments after each line that'd help show what each property is

File: V1Documentation.md [hunk 12]
```diff
@@ -536,11 +447,6 @@ Raygun.filterSensitiveData([creditCardDataRegex]);
 By default this is applied to the UserCustomData object only (legacy behavior). To apply this to any key-value pair, you can change the filtering scope:
 
 ```javascript
-// V2
-rg4js('setFilterScope', 'all'); // Filter any key in the payload
-rg4js('setFilterScope', 'customData'); // Just filter the custom data (default)
-
-// V1
 Raygun.setFilterScope('all');
```
Comment (fundead) [inline V1Documentation.md]: This also needs `Raygun.setFilterScope('customData');`

File: V1Documentation.md [hunk 13]
```diff
@@ -609,6 +502,55 @@ In Chrome, if the origin script tag and remote domain do not meet these requirem
...
+
+## AngularJS
+
+You can hook failed Ajax requests with $http in AngularJS by providing an Interceptor that sends to Raygun on error. One possible simple implementation:
+
+```javascript
+$httpProvider.interceptors.push(function($q, dependency1, dependency2) {
+  return {
+   'requestError': function(rejection) {
+       Raygun.send(new Error('Failed $http request', rejection));
```
Comment (fundead) [inline V1Documentation.md]: This looks like the same as the current docs, think we decided on `Raygun.send(new Error('msg'), { rejection: rejection });`

File: V1Documentation.md [hunk 14]
```diff
@@ -609,6 +502,55 @@ In Chrome, if the origin script tag and remote domain do not meet these requirem
...
+    },
+
+    'responseError': function(rejection) {
+       Raygun.send(new Error('Failed $http response', rejection));
```
Comment (fundead) [inline V1Documentation.md]: As above
